onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -593,6 +593,13 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
         "XNNPACK", {{"intra_op_num_threads", std::to_string(performance_test_config.run_config.intra_op_num_threads)}});
 #else
     ORT_THROW("Xnnpack is not supported in this build\n");
+#endif
+  } else if (provider_name_ == onnxruntime::kWebGpuExecutionProvider) {
+#ifdef USE_WEBGPU
+    session_options.AppendExecutionProvider(
+        "WebGPU", {{"intra_op_num_threads", std::to_string(performance_test_config.run_config.intra_op_num_threads)}});
```

Comment:
Out of interest why is this specified separately for WebGPU vs passing through and using the SessionOptions value (which is what we do with the xnnpack EP)?

Reply:
this should be updated (together with onnxruntime/test/onnx/main.cc in next PR)